src/m365/pa/commands/app/app-remove.ts

```diff
@@ -71,8 +89,16 @@ class PaAppRemoveCommand extends PowerAppsCommand {
     }
 
     const removePaApp = async (): Promise<void> => {
+      let endpoint;
+      if (args.options.asAdmin) {
+        endpoint = `${this.resource}/providers/Microsoft.PowerApps/scopes/admin/environments/${formatting.encodeQueryParameter(args.options.environmentName!)}/apps/${formatting.encodeQueryParameter(args.options.name)}?api-version=2017-08-01`;
+      }
+      else {
+        endpoint = `${this.resource}/providers/Microsoft.PowerApps/apps/${formatting.encodeQueryParameter(args.options.name)}?api-version=2017-08-01`;
+      }
```

Comment:
These endpoint URLs have a lot of common parts. Let's just use a single inline condition to add the admin scope and environment when needed (just like we do with `pa app get`).

Reply:
I think that single inline condition was there and it was changed according to the previous review